### DISCUSSION TITLE
How to set transparency of button card

### DISCUSSION THREAD

**Discussion by gioleuzzi** [custom-cards/button-card#830] [Q&A]
Hi,
I built a panel in HA with some button cards for my tablet. I set an image in backround but the cards remain white-coloured. 
I would know how to set transparency of such cards in order to see the backround. 
Could someone help me?
Tks😉

**Comment by SirRhor** [comment]
Try this:

`color: rgba(0,0,0,0.4)`

The 0.4 value controls transparency on the card's background if you use the section that controls the background.
If that does not work, then try to use a theme, but that will affect everything on HA. 
I use this one: [https://github.com/basnijholt/lovelace-ios-themes](url)
